webapi/src/main/scala/org/knora/webapi/messages/v2/routing/authenticationmessages/AuthenticationMessagesV2.scala

```diff
@@ -97,3 +101,8 @@ trait AuthenticationV2JsonProtocol extends DefaultJsonProtocol with NullOptions
     jsonFormat(LoginApiRequestPayloadV2, "iri", "email", "username", "password")
   implicit val SessionResponseFormat: RootJsonFormat[LoginResponse] = jsonFormat1(LoginResponse.apply)
 }
+
+trait AuthenticationV2Serialization {
+  implicit lazy val encLoginResponse: JsonEncoder[LoginResponse] = DeriveJsonEncoder.gen[LoginResponse]
+  implicit lazy val decLoginResponse: JsonDecoder[LoginResponse] = DeriveJsonDecoder.gen[LoginResponse]
+}
```

Comment:
is this common to have the codec in the trait rather than in the companion object? In any case, as a codec, this could be derived as one, rather than deriving encoder and decoder separately.

Reply:
This comes from Ivan. @subotic , do you have any reason why you put it into the trait instead of the companion objects? So far, we have all other codecs in the companion objects.

I replaced the encoder/decoder with a single codec.